dev/devicelab/bin/tasks/module_test.dart

```diff
@@ -218,11 +218,11 @@ Future<void> main() async {
       final String analyticsOutput = analyticsOutputFile.readAsStringSync();
       if (!analyticsOutput.contains('cd24: android-arm64')
           || !analyticsOutput.contains('cd25: true')
-          || !analyticsOutput.contains('viewName: build/bundle')) {
+          || !analyticsOutput.contains('viewName: assemble')) {
```

Comment:
cc @xster fyi: this requires updating the extraction logic for add2app tracking.

Reply:
What are the specific implications? That the pod integrated build phase for iOS and include_flutter.groovy now are making non-reentrant calls into assemble instead and all other calls into assemble are re-entrant (unless the user specifically types flutter assemble into CLI for some reason)?

Also surprised you didn't have to update module_test_ios.dart.